src/WP_Export_Query.php

```diff
@@ -262,7 +266,7 @@ private function category_where() {
 
 	private function attachments_for_specific_post_types( $post_ids ) {
 		global $wpdb;
-		if ( !$this->filters['post_type'] ) {
```

Comment:
In the WordPress admin, exporting a specific post type doesn't include attachments. We need to persist the current behavior here. If the `--with-attachments` flag is included, then we can include the attachments.